wireapp/wire-android-reloaded
# fix: wrong orientation when uploading avatar through camera

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Avatar picture appeared rotated on the preview picker screen when taken from the camera. The reason was that the uploaded picture was only being resampled (fixed rotation and resized) only after confirming the upload. Now the resampling is done on the preview step already instead. 

#### How to Test

Navigate to user's profile, select Take photo option and upload a new image. Check the rotation is correct on the preview step



### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
